src/Components/test/testassets/Components.TestServer/RazorComponents/Pages/Forms/FormOutsideBindingContextNoOps.razor

```diff
@@ -1,18 +0,0 @@
-﻿@using Microsoft.AspNetCore.Components.Forms
-
-<h2>Form outside a cascading binding context no-ops</h2>
-
-<EditForm EditContext="_editContext" method="POST" OnValidSubmit="() => _submitted = true" >
-    <AntiforgeryToken />
-    <input id="send" type="submit" value="Send" />
-</EditForm>
-
-@if (_submitted)
-{
-    <p id="pass">Form submitted!</p>
-}
-
-@code{
-    bool _submitted = false;
-    EditContext _editContext = new EditContext(new object());
-}
```

Comment:
Is there a case for disabling the root cascading value?

Right now, we'll bind data from the root, but I think there is a case to say "I never want to bind a form using SSR". 

For example if you are only interested in prerendering.

Reply:
We could have some config to control this, but I suspect we wouldn't need it any time soon. People can already just not use `[SupplyValueFromForm]` if they aren't interested in receiving form data, and we haven't heard demand for a way to disable `[SupplyValueFromQuery]`.